packages/components/psammead-brand/src/index.jsx

```diff
@@ -69,34 +65,25 @@ const StyledSpan = styled.span`
 const BrandSvg = styled.svg`
   display: block;
   height: ${SVG_HEIGHT};
-  width: ${SVG_WIDTH};
```

Comment:
Sorry, on second thoughts can we set `width` using the `SVG_HEIGHT` and the ratio value in the SVG object please